cluster/gce/util.sh

```diff
@@ -2553,15 +2600,92 @@ function kube-up() {
     write-cluster-name
     write-controller-config
     create-autoscaler-config
-    create-master
-    create-nodes-firewall
-    create-nodes-template
-    if [[ "${KUBE_CREATE_NODES}" == "true" ]]; then
-      # Windows nodes take longer to boot and setup so create them first.
-      create-windows-nodes
-      create-linux-nodes
+    if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+      echo "DBG: Generating shared CA certificates"
+      generate-shared-ca-cert
```

Comment:
shared cert generation should be in the caller function instead of in this kube-up()

Reply:
will re-org this section in follow-up optimized PR